### PR TITLE
 Changes, Fixes

### PR DESCRIPTION
**Pull request by SGZ-Creations**
- leaded glass being unlocked 3 times in the tech tree. Removed from laser-miners1 & glass-plated-panels. should now be unlocked only by vacuum-chamber

- Changed Technology requirements:

Energy-liquid. Removed utility packs, Increased multiplier, Decreased count

Planck. Decreased count of packs, Increased multiplier. removed Utillity packs.

ds-oil (Darkstar Oil): Removed utility packs, Increased multiplier, Decreased count.

bio-sample: chnaged Utility packs to Production packs, Increased multiplier, Decreased count.

Neuro miners 2-4: Added Neural science pack Removed Utility packs, Increased & Decreased count,

Alien weapons: Added Neural science pack.

Rocket-payloads: Added Neural packs.

Rocket-payloads2: Added Neural packs, Increased multiplier, Decreased Count.

### CODE REVIEW

File: prototype/technology.lua [hunk 1]
```diff
@@ -263,12 +263,12 @@ data:extend(
...
+                    count = 500,
                     ingredients = {
-                        { "automation-science-pack", 1 },
-                        { "logistic-science-pack", 1 },
-                        { "chemical-science-pack", 1 },
-                        { "utility-science-pack", 1 }
+                        { "automation-science-pack", 4 },
+                        { "logistic-science-pack", 3 },
+                        { "chemical-science-pack", 2 },
+                        { "prduction-science-pack", 1 }
```
Comment (H4ckerxx44) [inline prototype/technology.lua]: Typo

File: prototype/technology.lua [hunk 2]
```diff
@@ -1357,11 +1354,11 @@ data:extend(
                 unit = {
                     count = 500,
                     ingredients = {
-                        { "automation-science-pack", 1 },
-                        { "logistic-science-pack", 1 },
+                        { "automation-science-pack", 2 },
+                        { "logistic-science-pack", 2 },
```
Comment (H4ckerxx44) [inline prototype/technology.lua:1358]: Why were those changed to 2?

File: prototype/technology.lua [hunk 3]
```diff
@@ -1411,7 +1408,8 @@ data:extend(
                         { "logistic-science-pack", 1 },
                         { "chemical-science-pack", 1 },
                         { "production-science-pack", 1 },
-                        { "utility-science-pack", 1 }
+                        { "utility-science-pack", 1 },
+                        { "neural-science-pack", 2 }
```
Comment (H4ckerxx44) [inline prototype/technology.lua:1412]: that should be 1

File: prototype/technology.lua [hunk 4]
```diff
@@ -1460,13 +1458,14 @@ data:extend(
...
-                        { "logistic-science-pack", 1 },
-                        { "chemical-science-pack", 1 },
-                        { "production-science-pack", 1 },
-                        { "utility-science-pack", 1 },
+                        { "automation-science-pack", 3 },
+                        { "logistic-science-pack", 3 },
+                        { "chemical-science-pack", 3 },
+                        { "production-science-pack", 3 },
+                        { "utility-science-pack", 3 },
+                        { "neural-science-pack", 2 },
```
Comment (H4ckerxx44) [inline prototype/technology.lua:1468]: these all should be either all 1 or 4 like the tier of the miners suggest imo

**Review by H4ckerxx44 — CHANGES_REQUESTED** [review]
Made some comments

File: prototype/technology.lua (hunk 4 above)
Comment (SGZ-Creations) [reply prototype/technology.lua:1468]: No

have u seen
image

File: prototype/technology.lua (hunk 2 above)
Comment (SGZ-Creations) [reply prototype/technology.lua:1358]: have u seen
image

File: prototype/technology.lua (hunk 1 above)
Comment (SGZ-Creations) [reply prototype/technology.lua]: fixed

File: prototype/technology.lua (hunk 4 above)
Comment (H4ckerxx44) [reply prototype/technology.lua:1468]: Most things I haved looked at were consistenly 1 required science pack with various durations

File: prototype/technology.lua (hunk 2 above)
Comment (H4ckerxx44) [reply prototype/technology.lua:1358]: Yes, obviously they have been changed cause you changed them, the question is why you changed them. They should not have changed.

File: prototype/technology.lua (hunk 3 above)
Comment (SGZ-Creations) [reply prototype/technology.lua:1412]: nope it shouldn't

File: prototype/technology.lua (hunk 2 above)
Comment (SGZ-Creations) [reply prototype/technology.lua:1358]: Well then do it all yourself. Don't ask me to add PR if u just are being a dick about it.

**Comment by SGZ-Creations** [comment]
F this
